tests/integration/testIntegration.nim

```diff
@@ -65,6 +81,20 @@ twonodessuite "Integration tests", debug1 = false, debug2 = false:
     check request.ask.slots == 3'u64
     check request.ask.maxSlotLoss == 1'u64
 
+  # TODO: We currently do not support encoding single chunks
+  # test "node retrieves purchase status with 1 chunk":
+  #   let expiry = (await provider.currentTime()) + 30
+  #   let cid = client1.upload("some file contents").get
+  #   let id = client1.requestStorage(cid, duration=1.u256, reward=2.u256, proofProbability=3.u256, expiry=expiry, collateral=200.u256, nodes=2, tolerance=1).get
+  #   let request = client1.getPurchase(id).get.request.get
+  #   check request.ask.duration == 1.u256
+  #   check request.ask.reward == 2.u256
+  #   check request.ask.proofProbability == 3.u256
+  #   check request.expiry == expiry
+  #   check request.ask.collateral == 200.u256
+  #   check request.ask.slots == 3'u64
+  #   check request.ask.maxSlotLoss == 1'u64
```

Comment:
Instead of commenting out this test, could we increase the dataset size instead so that there's more than one Block being encoded?

Reply:
Thats what I did, look above the commented out test, it's the same but with several blocks.

Reply:
In fact, I left the name of the original test as is and left this one as the edge case we'd want to enable after adding support for it.